src/lib/structures/expr.ml

```diff
@@ -70,11 +70,9 @@ and quantified = {
   toplevel : bool;
   user_trs : trigger list;
   binders : binders;
-  (* These fields should be (ordered) lists ! important for skolemization *)
-  sko_v : t list;
+  sko_v : t list; (* This list has to be ordered for the skolemization. *)
```

Comment:
The original comment applies to *both* `sko_v` and `sko_vty` but the new comment only mentions `sko_v`.